### PR TITLE
e2e: fix

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: tests/e2e/demo.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+source /usr/local/sbin/demo-magic.sh
+
```
Comment (roytman) [inline tests/e2e/demo.sh:4]: you can consider adding `reset` here

File: tests/e2e/Makefile [hunk 2]
```diff
@@ -11,25 +11,49 @@ etcd:
...
-	go run $(ROOT_DIR)/pkg/cmd/client/client.go $(CLIENT_ARGS)
+	go run $(CLIENT_FILES) $(CLIENT_ARGS)
+
+INSTALL_TOOLS += /usr/bin/pv
+/usr/bin/pv:
+	sudo apt-get install pv
+
+INSTALL_TOOLS += /usr/bin/ovsdb-client
+/usr/bin/ovsdb-client:
+	sudo apt-get insall openvswtich-common
```
Comment (roytman) [inline tests/e2e/Makefile:51]: sudo apt-get install ovn-common -y

Comment (roytman) [inline tests/e2e/Makefile:51]: -y

Comment (hunchback) [reply tests/e2e/Makefile:51]: ok

Comment (hunchback) [reply tests/e2e/Makefile:51]: ok

**Comment by roytman** [comment]
does it fix `make server`?

File: tests/e2e/demo.sh (hunk 1 above)
Comment (hunchback) [reply tests/e2e/demo.sh:4]: ok

**Pull request merged**